scripts/docker/test/build-image_arm64.sh

```diff
@@ -0,0 +1,16 @@
+set -e
+
+arch=arm64
+
+image=arangodb/test-ubuntu
+docker build  --platform linux/$arch -t $image:latest-$arch --file arangodb-test.Dockerfile .
+
+OS_VERSION=$(docker run --rm -it $image:latest-$arch cat etc/os-release | grep -Po "VERSION_ID=\"\K\d+\.\d+")
```

Comment:
Shouldn't this be an absolute path to `/etc` here?
```suggestion
OS_VERSION=$(docker run --rm -it $image:latest-$arch cat /etc/os-release | grep -Po "VERSION_ID=\"\K\d+\.\d+")
```